scripts/dev_setup.py

```diff
@@ -21,11 +25,14 @@ def exec_command(command):
         print(err, file=sys.stderr)
         sys.exit(1)
 
+
 # install general requirements and azure-cli
 exec_command('python -m pip install --upgrade pip')
 
-if os.path.isfile('./requirements.txt'):
-    exec_command('pip install -r requirements.txt')
+# install to edge build of azure-cli
```

Comment:
Add the Env set to azure cli extension directory. Right now az cli keeps picking its extnesions from cliextensions instead of devcliextensions and hence our script won't work. 

I tired on linux and windows vm.